fog-core.gemspec

```diff
@@ -2,6 +2,7 @@
 lib = File.expand_path("../lib", __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require "fog/core/version"
+require 'english'
```

Comment:
I don't think english is supported by older versions of ruby which fog-core still supports. Also, double quotes.